server/README.md

```diff
@@ -64,7 +66,14 @@ Now you can install rails and bundler.
 gem install bundler 
 gem install rails
 ```
-
+Check your bundler version using
+```
+bundler --version
+```
+If you bundler version is not 2.1.4 , then install it using the following command
+```
+gem install bundler -v 2.1.4
```

Comment:
When someone uses the commands listed just before, `gem install bundler` is among them. That will install "latest released Bundler", which, according to [RubyGems.org](https://rubygems.org/gems/bundler) is 2.1.4.

So, the existing instructions should work, unchanged.

Reply:
Did they not work, for you?

Reply:
no , it did but a lot of questions are there on stackoverflow j just to avoid any confusion for beginners, i think they can verify easily

Reply:
it also some times depenfd on  system i am regular open source contributor so i do a lot of stuff in ruby

Reply:
@olleolleolle  sir i just want to help  others if they run into any error

Reply:
@olleolleolle you can suggest if there is some issue, but i felt it will be helpful. waiting for reply

Reply:
1. Avoid putting in exact version numbers. We don't have bandwidth to maintain a README with many version numbers in. They change.
2. The part where the user can verify the installation is very good. That inspires trust in the installation. Ruby version? RubyGems version?
3. It's not strictly required to use the latest released Bundler to work with this project.

Reply:
can i add command in my updates if a person needs to update ruby gem